shared/src/summarize_result.js

```diff
@@ -66,6 +66,11 @@ async function setup() {
   }
 
   summaryStatsTimeSavedElement.innerText = '0 minutes';
+  const summaryCloseElement = document.getElementById('close_summary');
+  if (!summaryCloseElement) {
+    console.error('Could not find summarize close element');
+    return;
+  }
```

Comment:
Can you please add a `summaryCloseElement.style.display = '';` after this? Similar to `summaryStatsElement`, it'll ensure that if the state is initialized in some other way in the future, it'll also reset the button's initial state.

Reply:
Do you mean `summaryCloseElement.style.display = 'none';`? That's how the `summaryStatsElement` is handled.

Reply:
Exactly!